gtk4/src/auto/string_list.rs

```diff
@@ -16,7 +16,7 @@ glib::wrapper! {
 
 impl StringList {
     #[doc(alias = "gtk_string_list_new")]
-    pub fn new(strings: &[&str]) -> StringList {
+    pub fn new(strings: Option<&[&str]>) -> StringList {
```

Comment:
those two don't make sense to have as nullable

Reply:
done